Modules/FindCUDA.cmake

```diff
@@ -750,7 +793,7 @@ else()
 endif()
 
 if(CUDA_USE_STATIC_CUDA_RUNTIME)
-  if(UNIX)
+  if(UNIX AND NOT ANDROID)
```

Comment:
CMake does not define an `ANDROID` value.  You could check `CMAKE_SYSTEM_NAME` instead.